package.json

```diff
@@ -11,7 +11,8 @@
     "@emotion/styled": "^11.8.1",
     "framer-motion": "^6.3.3",
     "react": "^18.1.0",
-    "react-dom": "^18.1.0"
+    "react-dom": "^18.1.0",
+    "typescript": "^4.6.4"
```

Comment:
There’s no typescript code in this project

Reply:
The reason i had to add this is, because I wasn't able to run the tests without it. If i recall some package had a dependency on it. I wonder if you may have typescript installed globally - or something else was acting up. 👍

Reply:
Oh, it looks like I do have it installed globally as a transitive dependency of another CLI tool. Interesting!